AlphaWallet/InCoordinator.swift

```diff
@@ -61,7 +62,13 @@ class InCoordinator: NSObject, Coordinator {
     private var eventSourceCoordinatorForActivities: EventSourceCoordinatorForActivities?
     private let coinTickersFetcher: CoinTickersFetcherType
     private lazy var eventSourceCoordinator: EventSourceCoordinatorType = createEventSourceCoordinator()
-    var tokensStorages = ServerDictionary<TokensDataStore>()
+    lazy var tokensDataStore: TokensDataStoreType = {
+        let tokensDataStore = TokensDataStore(realm: realm, account: wallet)
+        for each in config.enabledServers {
+            tokensDataStore.addEthToken(forServer: each)
```

Comment:
These seems to be done in 2 places in the codebase. Maybe we can move it into `TokensDataStore.init()` or somewhere central? Also have to be careful that it is added for all supported servers (not just enabled ones) and for custom ones when the user adds them

Reply:
yes, code has already updated